Formula/libzt.rb

```diff
@@ -0,0 +1,29 @@
+class Libzt < Formula
+  desc "ZeroTier: libzt -- An encrypted P2P networking library for applications"
+  homepage "https://www.zerotier.com"
+
+  url "https://github.com/zerotier/libzt.git",
+  :tag      => "1.3.0",
+  :revision => "2a377146d6124bb004b9aa263c47f7df2366e7ea"
+
+  depends_on "cmake" => :build
+  depends_on "make" => :build
+
+  def install
+    system "make", "update"
```

Comment:
What does this do?

Reply:
Made this more explicit by unpacking the commands and putting them in formula. This just pulled submodules and patched them before build.